### PR TITLE
feat: Add custom User-Agent

### PR DESCRIPTION
**Pull request by paultranvan**
Sends a custom User Agent when sending requests through the API service. This will notably help the support to be able to know the extension version.

### CODE REVIEW

File: src/browser/browserApi.ts [hunk 1]
```diff
@@ -117,6 +117,16 @@ export class BrowserApi {
         }
     }
 
+    static getApplicationName(): string {
+        if (BrowserApi.isChromeApi) {
+            return chrome.runtime.getManifest().name;
```
Comment (Crash--) [inline src/browser/browserApi.ts:122]: I know that the extension is called 'Cozy' on the Store, but for the UA, maybe we can have something more specific? Today `Cozy` calls only /bitwarden/ routes, but we don't know if it'll be the case in the future, and having something like `io.cozy.pass.browser` can be futur proof. 

Also, can you check with @sblaisot & @clochix I think we have few scripts that produce statistics based on the UA and these scripts want a specific shape for the UA.

Comment (sblaisot) [reply src/browser/browserApi.ts:122]: Currently statistics scripts recognize user agents in the form `io.cozy.<slug>-mobile` for mobile apps so I suggest using `io.cozy.pass-browser` here and adapt our statistics to handle `-browser` extension

Comment (paultranvan) [reply src/browser/browserApi.ts]: Sure, updated with 3d747fb68349c45ced6177d42bd5383020d95490

Comment (paultranvan) [reply src/browser/browserApi.ts]: Fixed with 31b73c0be087d47257b32d7106e8a13fd1eef9f0 , as pointed by https://github.com/cozy/cozy-keys-browser/pull/109#commitcomment-45655981

**Review by ptbrowne — APPROVED** [review]

**Review by Crash-- — APPROVED** [review]

**Pull request merged**